mail_private/__openerp__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+{
+    "name": """Select recipients for messages""",
+    "summary": """Module allow select recipients for send messages""",
```

Comment:
Allows you to send private messages to specified recipients only no matter who are in followers list.